XBVC/emitters/templates/c_interface.jinja2

```diff
@@ -65,10 +69,12 @@ float combine_float(struct splitfloat s)
 {
     float result_whole = (float)s.whole;
     float result_frac = 0;
+    uint32_t mul = 10;
```

Comment:
After talking with Jeff off line: I think this is a clever solution! Perhaps add a comment explaining what's going on here, tho, so that future slowpokes like me won't have to puzzle out why the decoding inverts the order of the digits.